### PR TITLE
Show appointment details on users activities

### PR DESCRIPTION
**Pull request by AndrewVos**
Let's try this again:

### CODE REVIEW

File: app/jobs/pusher_activity_notification_job.rb [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
     )
   end
 
   private
 
-  def render_activity(activity)
-    ApplicationController.new.render_to_string(
-      partial: activity
+  def render_activity(activity, details)
+    ApplicationController.renderer.render(
```
Comment (benlovell) [inline app/jobs/pusher_activity_notification_job.rb]: I think that can be reduced to `ApplicationController.render( ...`

File: spec/features/guider_views_their_activities_spec.rb [hunk 2]
```diff
@@ -34,11 +34,13 @@ def when_they_view_their_activity
   end
 
   def then_they_see_all_activities
+    first_id = @appointments.first.id
+    second_id = @appointments.second.id
     expect(@page.activities.map(&:text)).to match_array [
```
Comment (benlovell) [inline spec/features/guider_views_their_activities_spec.rb]: I wonder if this is too specific by matching exactly on the rendered text of each activity? The activity classes themselves seem to be the best place to spec the generated output. Maybe we should just verify the presence of entries here with known tags (ID or something)?

File: spec/features/user_views_booking_activities_spec.rb [hunk 3]
```diff
@@ -89,8 +89,8 @@ def when_somebody_else_adds_an_activity
   end
 
   def then_it_appears_dynamically
-    @page.activity_feed.wait_for_dynamically_loaded_activities(3)
-    activity = @page.activity_feed.dynamically_loaded_activities.first
+    expect(@page.activity_feed).to have_activities(count: 5, wait: 10)
```
Comment (benlovell) [inline spec/features/user_views_booking_activities_spec.rb]: I'd be curious to figure out this issue with needed to specify the wait time explicitly. Not a blocker for now though.

File: spec/features/guider_views_their_activities_spec.rb (hunk 2 above)
Comment (AndrewVos) [inline spec/features/guider_views_their_activities_spec.rb]: Regex

**Comment by AndrewVos** [comment]
@benlovell https://github.com/guidance-guarantee-programme/telephone_appointment_planner/pull/212/commits/00a3bb3bdc92de4f3071fd814ab98fb2565de507 <- sound good?

**Comment by benlovell** [comment]
@AndrewVos nice 💯

File: app/jobs/pusher_activity_notification_job.rb (hunk 1 above)
Comment (AndrewVos) [inline app/jobs/pusher_activity_notification_job.rb]: Done

File: spec/features/guider_views_their_activities_spec.rb (hunk 2 above)
Comment (AndrewVos) [inline spec/features/guider_views_their_activities_spec.rb]: @benlovell https://github.com/guidance-guarantee-programme/telephone_appointment_planner/pull/212/commits/5bda3a3deedf9fa6505473acffd675ad94859e36

**Pull request merged**